### PR TITLE
Updated Footer style

### PR DESCRIPTION
**Pull request by IlanKro**
Seems a little different in my browser than in Figma but I used the same settings (font/font size).
and I didn't link the pages yet, I didn't create the under construction page yet.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -280,16 +280,15 @@ <h2>Keep in touch</h2>
     </div>
   </div>
 
-  <div class="footer">
+  <div class="footer">
```
Comment (soryy708) [inline index.html]: Unnecesary whitespace change

File: index.html [hunk 2]
```diff
@@ -280,16 +280,15 @@ <h2>Keep in touch</h2>
...
-      </p>
-      <p>
-        <a href="http://www.freepik.com">Graphics by Freepik / macrovector</a> <a href="https://storyset.com/people">/
-          Freepik Storyset</a>
-      </p>
+      <img class="footerLogo" src="assets/whiteLogo.svg"/>           
+      <p class="credits">
+        Qint 2021. All rights reserved        
+      </p>      
+      <navbar>
```
Comment (soryy708) [inline index.html]: There's no such thing as `<navbar>` in strict HTML

File: style.css [hunk 3]
```diff
@@ -505,6 +513,60 @@ hr {
...
-.footer p {
+.footer {
+  background-color: #464646;
+}
+
+.footer p,
+a {
   margin: 0 0.25em;
+  color: #ffffff;
+  font-family: Al Nile;
```
Comment (soryy708) [inline style.css:355]: This font family is non-standard and doesn't work. Raphael needs to change it.

File: style.css [hunk 4]
```diff
@@ -505,6 +513,60 @@ hr {
...
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+}
+.footer .credits {
+  margin: auto;
+  order: 2;
```
Comment (soryy708) [inline style.css:362]: Why give things an explicit order?

File: style.css [hunk 5]
```diff
@@ -505,6 +513,60 @@ hr {
...
+}
+.footer navbar {
+  margin-right: auto;
+  margin-top: auto;
+  margin-bottom: auto;
+  flex-direction: row;
+  order: 3;
```
Comment (soryy708) [inline style.css:369]: Why give things an explicit order?

File: style.css [hunk 6]
```diff
@@ -505,6 +513,60 @@ hr {
...
+}
+.footerLogo {
+  margin-left: auto;
+  margin-top: auto;
+  margin-bottom: auto;
+  position: relative;
+  order: 1;
```
Comment (soryy708) [inline style.css:376]: Why give things an explicit order?

File: style.css [hunk 7]
```diff
@@ -505,6 +513,60 @@ hr {
...
+}
+
+.footer navbar span {
+  display: none;
+  color: #ffff;
+}
+
+@media screen and (max-width: 490px) {
```
Comment (soryy708) [inline style.css:384]: Why 490px?

File: style.css [hunk 8]
```diff
@@ -505,6 +513,60 @@ hr {
...
+  .footer navbar span {
+    display: inline;
+  }
+
+  .footer .content navbar {
+    flex-basis: 100%;
+    text-align: center;
+    order: 1;
```
Comment (soryy708) [inline style.css:392]: Why give things an explicit order?

File: style.css [hunk 9]
```diff
@@ -505,6 +513,60 @@ hr {
...
+  }
+
+  .footerLogo {
+    order: 2;
```
Comment (soryy708) [inline style.css:396]: Why give things an explicit order?

File: style.css [hunk 10]
```diff
@@ -505,6 +513,60 @@ hr {
...
+  }
+  .footer .credits {
+    text-align: end;
+    flex: max-content;
+    order: 3;
```
Comment (soryy708) [inline style.css:401]: Why give things an explicit order?

**Comment by soryy708** [comment]
A bunch of formatting changes were committed and are unnecesary. This causes merge conflicts for other people.
Review what code you're staging before committing to avoid this in the future.

File: hebrew.html [hunk 11]
```diff
@@ -296,15 +296,17 @@ <h2>שמרו איתנו על קשר</h2>
...
-        <a href="http://www.freepik.com">גרפיקה של Freepik / macrovector</a> <a href="https://storyset.com/people">/
-          Freepik Storyset</a>
-      </p>
+    <div class="content">  
+        <img class="footerLogo" src="assets/whiteLogo.svg"/>       
+        <p class="credits">
+          Qint 2021. כל הזכויות שמורות          
+        </p>       
+        <navbar>
+          <a>אבטחה</a><span class=dividor>|</span>
```
Comment (soryy708) [inline hebrew.html]: You're not styling the `dividor` class anywhere so why declare it?

File: index.html (hunk 2 above)
Comment (IlanKro) [reply index.html]: Changed to div with the class "navbar

File: style.css (hunk 3 above)
Comment (IlanKro) [reply style.css:355]: I used what I saw in Figma

File: style.css (hunk 4 above)
Comment (IlanKro) [reply style.css:362]: That's the solution I found for the problem.

File: style.css (hunk 7 above)
Comment (IlanKro) [reply style.css:384]: That's the point where the items don't fit in the same row.

File: hebrew.html (hunk 11 above)
Comment (IlanKro) [reply hebrew.html]: Oh right in the end I just used the span tag, it was initially a div with the class.

File: index.html (hunk 1 above)
Comment (IlanKro) [reply index.html]: weird I had 2 checkboxes near prettier when I uploaded.

Comment (IlanKro) [reply index.html]: Guess it doesn't style html

File: style.css (hunk 4 above)
Comment (IlanKro) [reply style.css:362]: https://stackoverflow.com/questions/40910851/change-div-position-depending-on-screen-resolution-flexbox

File: style-en.css [hunk 12]
```diff
@@ -13,3 +13,11 @@
     margin-left: 1.5em;
   }
 }
+.footer navbar {
```
Comment (soryy708) [inline style-en.css:16]: This doesn't select any elements because there's no `<navbar>` element anywhere

**Comment by soryy708** [comment]
https://github.com/Queueint/landing/pull/18

**Pull request closed without merging**